tests/test_mkdocs_config.py

```
@@ -0,0 +1,611 @@
+import pytest
+import os
+import tempfile
+import shutil
+import yaml
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)